Makefile.vc

```diff
@@ -0,0 +1,266 @@
+#
+# Options:
+# machine=Win32|Win64
+# prefix=\path\to\install
+#
+# If no machine is specified, the makefile uses the systems
+# PROCESSOR_ARCHITECTURE.
+#
+# Compiling:  nmake -f Makefile.vc
+# Installing: nmake -f Makefile.vc prefix=\path\to\x install
+#
+
+VERSION=0.1.5
+
+# Compiler, linker, librarian, and other tools
+CC  = cl
+CXX = cl
+LD  = link
+AR  = lib
+
+!IFDEF MACHINE
+!IF "$(MACHINE)" == "Win64"
+TARGET_FLAGS = /D__x86_64__
```

Comment:
Instead of adding a manual define of `__x86_64__` via the makefile, I would much rather extend the existing arch check in `libFDK/include/FDK_archdef.h`. Currently it does include detection of 32 bit x86 MSVC like this:

    (defined(_MSC_VER) && defined(_M_IX86))

So I'd rather add another similar condition at the end:

    (defined(_MSC_VER) && defined(_M_X64))